Source/Scene/Scene.js

```diff
@@ -2880,6 +2880,7 @@ Scene.prototype.updateAndExecuteCommands = function (
     mode !== SceneMode.SCENE2D ||
     this._mapMode2D === MapMode2D.ROTATE
   ) {
+    exitWebVRUpdateViewport(this, passState);
```

Comment:
I'm not that familiar with VR but having this code run per-tick does not seem correct to me.

Reply:
The current bug problem is that when entering VR state and exiting again, `viewer.scene.pick(event.position); ` can't pick up the relevant entity  object, and the return value is undefined.


The code I modified is to restore the operation done when entering VR, corresponding to the exitWebrupdateViewPort method.